docs/content/docs/development/query-table.md

```diff
@@ -110,6 +110,32 @@ SELECT * FROM MyTable$snapshots;
 By querying one table's snapshots table, you can know the commit and expiration
 information about that table and time travel through the data.
 
+## Schemas Table
+
+You can query the historical schemas of the table through Flink SQL.
+
+```sql
+SELECT * FROM MyTable$schemas;
+
++-----------+----------------------------------------+----------------+--------------+-----------+-----------+
+| schema_id |                  fields                | partition_keys | primary_keys |  options  | commonts  |
++-----------+----------------------------------------+----------------+--------------+-----------+-----------+
+|   1       | [{"id":0,"name":"a","type":"BIGINT"... |     ["k1"]     |  ["k1", "k2] |    {}     | my table  |
```

Comment:
In SQL-CLI, do we have option to show long field? I think we can use it here.
`sql-client.display.max-column-width`

Reply:
> In SQL-CLI, do we have option to show long field? I think we can use it here. `sql-client.display.max-column-width`

The config `sql-client.display.max-column-width` is supported in `streaming` only, I have created a issue https://issues.apache.org/jira/browse/FLINK-30303 in `Flink`